src/controllers/bulk/bulkcontroller.cpp

```diff
@@ -5,9 +5,15 @@
 #include "controllers/bulk/bulksupported.h"
 #include "controllers/defs_controllers.h"
 #include "moc_bulkcontroller.cpp"
+#include "util/cmdlineargs.h"
 #include "util/time.h"
 #include "util/trace.h"
 
+namespace {
+using namespace std::literals;
+constexpr std::seconds kBulkSendTimeout = 5s;
```

Comment:
I think it is: 
```
    using namespace std::literals::chrono_literals; 
    constexpr std::chrono::seconds kBulkSendTimeout = 5s;
```
However the struggle here, confirms IMHO that using 5s is a bit over engineered here when not using this for chronio functions. So I would release the compiler from some work and use use the libusb type without a cast:
```
constexpr unsigned int kBulkSendTimeoutMs = 5000;
```
The Ms prefix documents the value unit. 
Both is fine for me.

Reply:
I disagree honestly. The added compile time is negligible. This is about avoiding [primitive obsession](https://refactoring.guru/smells/primitive-obsession).

Reply:
The resource you linked make sense, but in our case, we don't own the API and simply must comply to libusb asking for a tasteless `uint`.
To be honest, I'm leaning towards @daschuer's suggestion as I find it more readable - `std::milliseconds(kBulkSendTimeout).count()` feels a bit cryptic to me (had to go to the std lib to see what `count` was doing). Appreciate you are trying to roll out std on time within the Mixxx API tho so can't make up my mind. 
Would a @mixxxdj/developers help us with the tie here?

Reply:
In general I'm in favor for these literal types, and we should all get common to this std C++ syntax, as libraries like Qt convert their timing types towards this standard.
In this case, the literal type give us no type safety, as it's converted to the raw integer type before use.
Regarding readability, where a const value is only used once in the whole code - inline with the unit as comment is IMHO the most readable style:
```
c_function(5000, // Send timeout in milliseconds
           otherArgument); 
```

Reply:
Thanks @JoergAtGithub , that does sound like a great middle ground. @daschuer @Swiftb0y happy with that option?

Reply:
>  In this case, the literal type give us no type safety, as it's converted to the raw integer type before use.

Yeah thats unfortunate, but the conversion is done at the API boundary (so its like calling `.get()` on a unique_ptr when passing to a C API. I still prefer my original solution (an unnamed constant is not much better), but I won't insist on it. Just my two cents for what is best in the general case. This is only a one-off so as you rightfully pointed out, it doesn't make much difference.

Reply:
As I already mentioned, all suggestions are fine for me as well. 

I can confirm the original request of @Swiftb0y to replace a comment by code.

Comment difficult code, but avoid comments by making the code itself expressive when possible.